keycloak-event-emitter/src/main/java/io/cloudtrust/keycloak/kafkaeventemitter/KafkaEventCallback.java

```diff
@@ -0,0 +1,23 @@
+package io.cloudtrust.keycloak.kafkaeventemitter;
+
+import org.apache.kafka.clients.producer.Callback;
+import org.apache.kafka.clients.producer.ProducerRecord;
+import org.apache.kafka.clients.producer.RecordMetadata;
+import org.jboss.logging.Logger;
+
+public class KafkaEventCallback implements Callback {
+    private static final Logger logger = Logger.getLogger(KafkaEventCallback.class);
+
+    private final ProducerRecord record;
+    KafkaEventCallback(ProducerRecord record){
+        this.record = record;
+    }
+
+    @Override
+    public void onCompletion(RecordMetadata recordMetadata, Exception e) {
```

Comment:
You are implementing Callback interface with only one method... can't it be a lambda expression?

Reply:
Yes, it is better. I will make the change